Modules/Module_Processing/Module_ASL/xASL_wrp_VisualQC_ASL.m

```diff
@@ -134,12 +134,13 @@ function xASL_wrp_VisualQC_ASL(x)
     if xASL_exist(x.P.Path_c1T1,'file') && xASL_exist(x.P.Path_c2T1,'file')
         % If segmented files exist, then the PVgm should already be prepared previously
 
-        if ~xASL_exist(x.P.Path_PWI4D, 'file')
+        if ~isfield(x.P, 'Path_PWI4D_used') || ~xASL_exist(x.P.Path_PWI4D_used, 'file')
```

Comment:
Thinking about it, this whole if statement is suboptimal, because you are catching something that should have happened before, and trying to decide in a detailed code part what we should have done before.

So: we should prefer a pipeline crash over sacrificing reliability

Reply:
Good point. I was merely updating the code, not inspecting it further. But you are right that fixing this might create confusion, while in practice, the file should be always there unless things went wrong.

So I'm deleting this whole update. Done